test/functional/editor/undo_spec.lua

```diff
@@ -18,6 +17,10 @@ local function lastmessage()
   return messages[#messages]
 end
 
+local feed_undo = function(level)
+  feed(':undo!' .. (level and (' ' .. level) or '') .. '<CR>')
```

Comment:
If possible, `n.command` is preferred (that's why `feed_command` is deprecated). Can that be used? `feed()` is only needed for cases where asynchronous behavior is wanted.

Maybe *some* of these tests need async, but the others should use `n.command`.